spiceaidocs/docs/cli/reference/connect.md

```diff
@@ -1,28 +1,34 @@
 ---
-title: "connect"
-sidebar_label: "connect"
-pagination_prev: null
-pagination_next: null
----
+ title: "connect"
+ sidebar_label: "connect"
+ pagination_prev: null
+ pagination_next: null
+ ---
 
-Connect to Spice.ai Cloud Platform App
+ Connect to an app on the Spice.ai Cloud Platform.
```

Comment:
Make point that explains difference with `spice add` (i.e. connecting to spice in Cloud platform, vs adding components into local spicepod).